app/frontend/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,42 @@
+
+.navbar {
+  margin-left: 300px;
+  display: flex;
+  // justify-content: space-around;
+  align-items: center;
```

Comment:
Remove the comment please.
And don't use a fixed margin because this is not reactive at all. Use justify-content: center instead when you want to center something on the x axis

Reply:
Sure! My bad